streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -710,10 +712,10 @@ public StreamThread(final Time time,
         this.originalReset = originalReset;
         this.versionProbingFlag = versionProbingFlag;
 
-        this.pollTimeMs = config.getLong(StreamsConfig.POLL_MS_CONFIG);
+        this.pollTime = Duration.ofMillis(config.getLong(StreamsConfig.POLL_MS_CONFIG));
```

Comment:
as above: `"main.consumer.poll.ms"` ?

Reply:
I guess not, following Guozhang's comment.